test/__snapshots__/ConfigTestCases.basictest.js.snap

```diff
@@ -1897,7 +1897,7 @@ exports[`ConfigTestCases css css-modules exported tests should allow to create c
tScope/a:\\\\.\\\\/style\\\\.module\\\\.css-a/animationName:\\\\.\\\\/style\\\\.module\\\\.css-animationName/b:\\\\.\\\\/style\\\\.module\\\\.css-b/c:\\\\.\\\\/style\\\\.module\\\\.css-c/d:\\\\.\\\\/style\\\\.module\\\\.css-d/animation-name:--\\\\.\\\\/style\\\\.module\\\\.css-animation-name/mozAnimationName:\\\\.\\\\/style\\\\.module\\\\.css-mozAnimationName/my-color:--\\\\.\\\\/style\\\\.module\\\\.css-my-color/c0ffee:\\\\.\\\\/style\\\\.module\\\\.css-c0ffee/padding-sm:\\\\.\\\\/style\\\\.module\\\\.css-padding-sm/padding-lg:\\\\.\\\\/style\\\\.module\\\\.css-padding-lg/nested-pure:\\\\.\\\\/style\\\\.module\\\\.css-nested-pure/nested-media:\\\\.\\\\/style\\\\.module\\\\.css-nested-media/nested-supports:\\\\.\\\\/style\\\\.module\\\\.css-nested-supports/nested-layer:\\\\.\\\\/style\\\\.module\\\\.css-nested-layer/not-selector-inside:\\\\.\\\\/style\\\\.module\\\\.css-not-selector-inside/nested-var:\\\\.\\\\/style\\\\.module\\\\.css-nested-var/again:\\\\.\\\\/style\\\\.module\\\\.css-again/nested-with-local-pseudo:\\\\.\\\\/style\\\\.module\\\\.css-nested-with-local-pseudo/local-nested:\\\\.\\\\/style\\\\.module\\\\.css-local-nested/bar:\\\\.\\\\/style\\\\.module\\\\.css-bar/id-foo:\\\\.\\\\/style\\\\.module\\\\.css-id-foo/id-bar:\\\\.\\\\/style\\\\.module\\\\.css-id-bar/nested-parens:\\\\.\\\\/style\\\\.module\\\\.css-nested-parens/local-in-global:\\\\.\\\\/style\\\\.module\\\\.css-local-in-global/in-local-global-scope:\\\\.\\\\/style\\\\.module\\\\.css-in-local-global-scope/class-local-scope:\\\\.\\\\/style\\\\.module\\\\.css-class-local-scope/class-in-container:\\\\.\\\\/style\\\\.module\\\\.css-class-in-container/deep-class-in-container:\\\\.\\\\/style\\\\.module\\\\.css-deep-class-in-container/placeholder-gray-700:\\\\.\\\\/style\\\\.module\\\\.css-placeholder-gray-700/placeholder-opacity:--\\\\.\\\\/style\\\\.module\\\\.css-placeholder-opacity/test:--\\\\.\\\\/style\\\\.module\\\\.css-test/baz:--\\\\.\\\\/style\\\\.module\\\\.css-baz/slidein:\\\\.\\\\/style\\\\.module\\\\.css-slidein/my-global-class-again:\\\\.\\\\/style\\\\.module\\\\.css-my-global-class-again/first-nested:\\\\.\\\\/style\\\\.module\\\\.css-first-nested/first-nested-nested:\\\\.\\\\/style\\\\.module\\\\.css-first-nested-nested/first-nested-at-rule:\\\\.\\\\/style\\\\.module\\\\.css-first-nested-at-rule/first-nested-nested-at-rule-deep:\\\\.\\\\/style\\\\.module\\\\.css-first-nested-nested-at-rule-deep/foo:--\\\\.\\\\/style\\\\.module\\\\.css-foo/\\\\.\\\\/style\\\\.module\\\\.css,myCssClass:\\\\.\\\\/style\\\\.module\\\\.my-css-myCssClass/\\\\.\\\\/style\\\\.module\\\\.my-css,\\\\.\\\\/style\\\\.module\\\\.css\\\\.invalid,UnusedClassName:\\\\.\\\\/identifiers\\\\.module\\\\.css-UnusedClassName/variable-unused-class:--\\\\.\\\\/identifiers\\\\.module\\\\.css-variable-unused-class/UsedClassName:\\\\.\\\\/identifiers\\\\.module\\\\.css-UsedClassName/variable-used-class:--\\\\.\\\\/identifiers\\\\.module\\\\.css-variable-used-class/\\\\.\\\\/identifiers\\\\.module\\\\.css;}"
```

Comment:
This is getting too big, I'm thinking about use something like LZW to compress it in the future PR

Reply:
<img width="1348" alt="Screenshot 2024-02-21 at 18 42 36" src="https://github.com/webpack/webpack/assets/42857895/e1b7a986-fe66-4523-b96d-f623d00f1e5b">

Reply:
Sounds good for me

Reply:
Maybe under the option